packages/gatsby-telemetry/src/store.js

```diff
@@ -7,84 +6,42 @@ const {
   existsSync,
   unlinkSync,
 } = require(`fs`)
-const { ensureDirSync } = require(`fs-extra`)
-const Configstore = require(`configstore`)
 
 module.exports = class Store {
-  config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
-
-  constructor() {}
-
-  getConfig(key) {
-    if (key) {
-      return this.config.get(key)
-    }
-    return this.config.all
-  }
-
-  updateConfig(...fields) {
-    this.config.set(...fields)
+  constructor(parentFolder) {
```

Comment:
I would rename this into something else. I don't really know a good word but I was thinking into baseDir or something. parentFolder is a weird name as a variable here.
```suggestion
  constructor(baseDir) {
```

Reply:
Good point!